src/SDKs/CognitiveServices/dataPlane/Vision/FormRecognizer/FormRecognizer/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Cognitive Services Form Recognizer SDK")]
+[assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services Form Recognizer APIs.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Please set version to `0.8.0-preview`

Reply:
I get following build error:
Properties\AssemblyInfo.cs(10,28,10,43): error CS7034: The specified version string does not conform to the required format - major[.minor[.build[.revision]]]

Reply:
In the `AssemblyInfo.cs` this should be set to `0.8.0.0`

Reply:
done.

Reply:
done.